impl/src/add_assign_like.rs

```diff
@@ -29,7 +29,7 @@ pub fn expand(input: &DeriveInput, trait_name: &str) -> TokenStream {
 
     quote! {
         #[automatically_derived]
-        impl #impl_generics ::core::ops::#trait_ident for #input_type #ty_generics #where_clause {
+        impl #impl_generics ::derive_more::core::ops::#trait_ident for #input_type #ty_generics #where_clause {
```

Comment:
We're re-exporting these traits now right. Any reason we cannot do:

```suggestion
        impl #impl_generics ::derive_more::#trait_ident for #input_type #ty_generics #where_clause {
```

Reply:
@JelteF I've stripped redundant paths where possible.